add compiled linuxarmv6l socket.io-client libs

mstllc opened the pull request.

Commit:
add compiled linuxarmv6l socket.io-client libs

gabrielstuff commented:
Thanks for the pull request. Could you add an example ? I guess it is for iOS ?

mstllc commented:
The compiled libs I added are for linuxarmv6l, or Raspberry Pi.  I don't think this pull request requires it's own example.  The example you already include will work on Raspberry Pi with the compiled libs I've added.

gabrielstuff commented:
I let @hogogil check and merge ! Thanks anyway running on the pi is a great
news !
Le mer. 20 avr. 2016 à 06:54, Max Thorson notifications@github.com a
écrit :

> The compiled libs I added are for linuxarmv6l, or Raspberry Pi. I don't
> think this pull request requires it's own example. The example you already
> include will work on Raspberry Pi with the compiled libs I've added.
> 
> —
> You are receiving this because you commented.
> 
> Reply to this email directly or view it on GitHub
> https://github.com/soixantecircuits/ofxSocketIO/pull/2#issuecomment-212256828
